### DISCUSSION TITLE
Add support for shortcuts with alternative keyboard layouts

### DISCUSSION THREAD

**Discussion by Frfri1** [ppy/osu#19257] [Ideas] [answered]
I use an AZERTY keyboard instead of a QWERTY keyboard, but the shortcuts aren't taking that into account. (azerty has a and q reversed, and z and w reversed). While searching a beatmap, I need to do ctrl+q to select all. In the editor, there is the same thing with ctrl+q, plus having to press ctrl+w to undo. 

It would be very nice to have this implemented, and those two shortcuts are already working like that in osu!stable.

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/issues/11143
